docs/application-security/intro.md

```diff
@@ -9,8 +9,28 @@ When using Azure and Github actions, care should be taken to not run github acti
 There is a risk if any of the App service configuration values are sensitive because they would be printed into the github actions log if the workflow is run in debug mode.
 
 #### Mitigations
+The only 100% effective way of mitigating the leaking of sensitive values is to use GitHub's log masking functionality. This can be used in conjunction with Azure key Vault.
```

Comment:
```suggestion
The only way of mitigating the leaking of sensitive values in the GitHub logs is to use GitHub's log masking functionality. This can be used in conjunction with Azure key Vault.
```